portlets/src/main/webapp/vue-app/realizations/components/Realizations.vue

```diff
@@ -40,7 +40,6 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
       :sort-desc.sync="sortDescending"
       disable-pagination
       hide-default-footer
-      must-sort
```

Comment:
to keep it as it is to make sure to have the sort option always displayed to end user (default sort = 'date')